app/interfaces/api/helpers/search_result_helpers.rb

```diff
@@ -95,4 +95,18 @@ def up_to_and_inc_pcmh_transfer?
   def last_injection_attempt_succeeded
     object&.last_injection_succeeded || false
   end
+
+  def scheme_12_warning
```

Comment:
https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3478/files#r476385241

Reply:
👍 have amended naming to scheme_12_fees_warning